.github/workflows/ScopeTesting.yml

```diff
@@ -9,9 +9,19 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Testing for macOS
+    - name: Testing for macOS (only on push)
+      if: github.event_name == 'push'
+      uses: undefinedlabs/scope-for-swift-action@v1
+      with:
+        dsn: ${{ secrets.SCOPE_DSN }}
```

Comment:
I believe the secret was removed from the repo (https://github.com/open-telemetry/opentelemetry-swift/pull/30/files). Or it refers to another secret?

Reply:
Yes, but I added it again ( with the same value that is published) I could not think any other way a fork can use its own value in its own commits and use the official one when doing a PR. The root problem is that Pull Requests in GitHub actions don’t inherit secrets from the repo where they are creating it, so need to use its own value for every commit, and use the global only on PR